Game/Source/GDKTestGyms/Private/RPCTimeoutPC.cpp

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+
+#include "RPCTimeoutPC.h"
+
+
+#include "Chaos/PhysicalMaterials.h"
+#include "Components/TextRenderComponent.h"
+#include "Engine/StreamableManager.h"
+#include "GameFramework/Character.h"
+#include "UObject/ConstructorHelpers.h"
+
+ARPCTimeoutPC::ARPCTimeoutPC()
+{
+	//Choose materials which belong to the Engine. This is in anticipation of the possibility of moving this test to the UnrealGDK plugin in the future.
+
+	const TCHAR* FailedMaterialPathString = TEXT("Material'/Engine/EngineDebugMaterials/VertexColorViewMode_RedOnly.VertexColorViewMode_RedOnly'");
+	static ConstructorHelpers::FObjectFinder<UMaterial>FailedMaterialFinder(FailedMaterialPathString);
+	FailedMaterialAsset = FailedMaterialFinder.Object;
+	checkf(IsValid(FailedMaterialAsset), TEXT("Could not find failed material asset %ls"), FailedMaterialPathString);
+	
+	const TCHAR* SoftMaterialPathString = TEXT("Material'/Engine/Tutorial/SubEditors/TutorialAssets/Character/TutorialTPP_Mat.TutorialTPP_Mat'");
+	SoftMaterialPath = FSoftObjectPath(SoftMaterialPathString);
```

Comment:
Why do we store this as a member variable?

Reply:
I would suggest writing code as its used, not how it can be used, it can always be changed later.

Reply:
fair enough, fixed